modules/dcache-nfs/src/main/java/org/dcache/chimera/nfsv41/door/NFSv41Door.java

```diff
@@ -858,6 +858,34 @@ public String call() throws IOException {
         }
     }
 
+
+    @Command(name = "show transfers filtered", hint = "show active transfers filtered",
+            description = "Show active transfers excluding proxy-io filtered by pool, client or pnfsid.")
+    public class ShowTransfersFilteredCmd implements Callable<String> {
+
+        @Argument(required = false, metaVar = "pool")
+        String pool;
```

Comment:
While exact matching Strings may be OK under some circumstances, client identifiers and often pool names are structured.  Therefore, it could be useful to support selecting a family of pools or a family of clients, rather than one in particular.

This is pretty easy to do: dCache already has a class to support globbing patterns (Glob).  The change would look something like:

    @Option(name="pool")
    Glob pool;

    // ...
        .filter( d -> pool == null ? true : pool.matches(d.getPool()))